site/content/en/docs/manual/advanced/annotation-with-skeletons/auto-skeletons.md

```diff
@@ -0,0 +1,61 @@
+---
+title: 'Automatic annotation with Skeletons'
+linkTitle: 'Automatic annotation with Skeletons'
+weight: 2
+description: 'Automatic annotation with Skeletons'
+---
+
+In this guide, we delve into the efficient process of annotating complex
+structures through the implementation of **Skeleton** automatic annotations.
+
+See:
+
+- [Creating auto annotation Skeleton task](#creating-auto-annotation-skeleton-task)
+- [Automatic annotation with Skeletons](#automatic-annotation-with-skeletons)
+  - [Editing skeletons on the canvas](#editing-skeletons-on-the-canvas)
+
+## Creating auto annotation Skeleton task
```

Comment:
I believe it should be described somewhere in labels constructor documentation.
It is just one more alternative way to add a label (any label, skeleton is only particular case) to the task. And it is not really related to skeletons automatic annotations